boards/arm/nucleo_h753zi/doc/index.rst

```diff
@@ -121,6 +121,9 @@ features:
 +-----------+------------+-------------------------------------+
 | SPI       | on-chip    | spi                                 |
 +-----------+------------+-------------------------------------+
+| USB       | on-chip    | usb_device                          |
```

Comment:
```suggestion
| USB OTG FS| on-chip    | USB device                          |
```

Reply:
As above, I don't think this change is necessary.